charts/flyte/templates/common/secret-auth.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: flyte-propeller-auth
+  name: flyte-common-auth
   namespace: {{ template "flyte.namespace" . }}
 type: Opaque
 stringData:
```

Comment:
@EngHabu  Why do we have static value here ? I think we should client_secret from values.yaml